src/uct/api/v2/uct_v2.h

```diff
@@ -616,19 +616,22 @@ typedef struct uct_ep_is_connected_params {
 
     /**
      * Device address to check for connectivity.
-     * This field must not be passed if iface_attr.dev_addr == NULL.
+     * This field must not be passed if address entry does not contain device
```

Comment:
Maybe: `this field must only be passed if...`

Reply:
what is "address entry" in the context of UCT API?

Reply:
then how to explain when each address is needed?